infra/modules/ecs_service/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "container_image" {
+  type = string
+}
+
+variable "service_name" {
+  type = string
+}
+
+variable "environment" {
+  type    = map(string)
+  default = {}
+}
+
+variable "secrets" {
+  type    = map(string)
+  default = {}
+}
+
+variable "container_port" {
+  type    = number
+  default = 9001
+}
```

Comment:
Nit: do we want to set a default here? Feels like there's an easy mistake where you forget to set it and wonder why you don't see anything.